File: sum_of_primes/SumOfPrimes.java
import java.util.ArrayList;

public class SumOfPrimes {
    public static void main (String[] args) {
        int sum = 0;
        int numberOfPrimes = 0;
        int iterator = 2;

        while (numberOfPrimes < 1000) {
            if (isPrime(iterator)) {
                sum += iterator;
                numberOfPrimes += 1;
            }
            iterator++;
        }

        System.out.println(sum);
    }

    private static boolean isPrime (int number) {
        if (number == 0 || number == 1) {
            return false;
        }
        else if (number == 2) {
            return true;
        }
        else {
            for (int i = 2; i < number; i++) {
                if (number % i == 0) {
                    return false;
                }
            }
        }

        return true;
    }
}

File: sum_of_primes/sum_of_primes.js
var upperLimit = 1000;

sum_of_primes(upperLimit)

function sum_of_primes(upperLimit) {
    var sumOfPrimes = 2;
    var x = 3;
    var primesCounter = 2;

    while (primesCounter <= upperLimit) {
        if (is_prime(x)) {
            sumOfPrimes += x;
            primesCounter++;
        }
        x++;
    }

    console.log(sumOfPrimes);
}

function is_prime(number) {
    for (var i = 2; i < number; i++) {
        if (number % i == 0) {
            return false;
        }
    }

    return true;
}